keyboards/boardwalk/keymaps/via/rules.mk

```
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Let's add LTO_ENABLE here too!

```suggestion
LTO_ENABLE = yes
```